website/docs/content/voice-and-tone/index.md

```diff
@@ -1,6 +1,30 @@
 ---
 title: Voice and Tone
-caption: Applications should be usable and accessible for all users of differing abilities.
-description: Applications should be usable and accessible for all users of differing abilities.
-previewImage: assets/illustrations/foundations/accessibility.jpg
+caption: How HashiCorp communicates with our users.
```

Comment:
This and the caption are the same except for one word, should probably one way or the other. Personally, I'd go with:

```suggestion
caption: How HashiCorp communicates with its users.
```